src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -466,7 +466,7 @@ private void scrollAndHighlightText(Text targetText) {
         }
 
         // type-casted to Hbox as every Text in helpWindow is wrapped by a Hbox as its parent
-        HBox box = (HBox) targetText.getParent().getParent().getParent();
+        HBox box = (HBox) targetText.getParent().getParent().getParent().getParent().getParent();
```

Comment:
would be good to change the comment above to explain the chains of getParent() for easier understanding of the code and reason for typecasting

Reply:
Great catch! I have added comments on why there is a huge chain now.